recipe/meta.yaml

```diff
@@ -11,49 +11,56 @@ source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: {{ sha256 }}
 
-
 build:
-  number: 1002
-  script: python -m pip install --no-deps --ignore-installed .  --verbose
+  number: 0
+  script:
+    - export SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}    # [unix]
+    - set SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}    # [win]
+    - {{ PYTHON }} -m pip install . --no-deps -vv
```

Comment:
```suggestion
    - SETUPTOOLS_SCM_PRETEND_VERSION={{ version }} {{ PYTHON }} -m pip install . --no-deps -vv
```

Reply:
This doesn't work. A lesson I've learned here: https://github.com/conda-forge/gstools-feedstock/commit/37bdbcdcf981f885c8037edd9689134d65ee97a3
This only works for `noarch` builds.

Reply:
Aww OK.